docs/source/status.rst

```diff
@@ -20,8 +20,9 @@ Implementation Status
 =====================
 
 The following tables summarize the features available in the various official
-Arrow libraries.  Unless otherwise stated, the Python, R, Ruby and C/GLib
-libraries follow the C++ Arrow library.
+Arrow libraries. All libraries currently follow version 1.0.0 of the Arrow
+format. See :ref:`status` for details about versioning. Unless otherwise stated,
```

Comment:
Is this supposed to be creating a link to itself or am I misunderstanding this?

Reply:
```suggestion
format. See :doc:`status` for details about versioning. Unless otherwise stated,
```